openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/edgefilters/core/MaximumSpeedCoreEdgeFilter.java

```diff
@@ -0,0 +1,49 @@
+/*  This file is part of Openrouteservice.
+ *
+ *  Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ *  GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ *  of the License, or (at your option) any later version.
+
+ *  This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Lesser General Public License for more details.
+
+ *  You should have received a copy of the GNU Lesser General Public License along with this library;
+ *  if not, see <https://www.gnu.org/licenses/>.
+ */
+package org.heigit.ors.routing.graphhopper.extensions.edgefilters.core;
+
+import com.graphhopper.routing.profiles.DecimalEncodedValue;
+import com.graphhopper.routing.util.EdgeFilter;
+import com.graphhopper.util.EdgeIteratorState;
+import com.graphhopper.routing.util.FlagEncoder;
+import org.heigit.ors.config.AppConfig;
+
+/**
+ * This class includes in the core all edges with speed more than the one set in the app.config file max_speed.
+ *
+ * @author Athanasios Kogios
+ */
+
+public class MaximumSpeedCoreEdgeFilter implements EdgeFilter {
+    private double maxSpeed =  ((AppConfig.getGlobal().getServiceParameter("routing.profiles.default_params","maximum_speed_lower_bound")) != null)
```

Comment:
The handling of `maximum_speed_lower_bound` value from the config file still needs some refactoring. Currently there are 3 different places over the code where this value is parsed and if missing the hard-coded value of 80 is used. The default value, even if hard-coded, should appear only once in the whole codebase.

I suggest to use `RouteProfileConfiguration` class to store the value of `maximumSpeedLowerBound`.  Have a look at the implementation of the `maximum_distance` parameter, it will be probably quite similar.

This change might require , for example, taht `maxSpeed` value is resolved upstream in `ORSGraphHopper` from `CmdArgs` set in `RoutingProfile`,  and passed to `MaximumSpeedCoreEdgeFilter` as a constructor argument.

Reply:
Update: please use `GraphProcessContext` rather than `CmdArgs` for passing the `maximum_speed_lower_bound`.